docs/3.0.0-beta.x/admin-panel/deploy.md

```diff
@@ -34,23 +34,23 @@ You might want to change the path to access to the administration panel. Here th
 
 It's very common to deploy the front-end and the back-end on different servers. Here are the required configurations to handle this case:
 
-**Path —** `./config/environment/**/server.json`.
+**Path —** `./config/server.js`.
 
 ```js
-{
-  "host": "localhost",
-  "port": 1337,
-  "cron": {
-    "enabled": false
+module.exports = {
```

Comment:
Removal of proxy block also needs to be added to migration guide (and I will need to update one-clicks for this change)

Reply:
Proxy is not removed in this PR. We just don't need to set it by default.

Reply:
Cool, yeah got my wires crossed between this and the other PR